packages/dashboard/src/project/hook/editHookEvents.tsx

```diff
@@ -12,32 +19,52 @@ interface EditHookEventsProps {
 }
 export const EditHookEvents = ({ hook, disabled }: EditHookEventsProps) => {
   const { errors, control } = useFormContext();
+
   return (
-    <Cell xs={12}>
+    <Grid item xs={12}>
```

Comment:
All `<Grid item>`s should be wrapped with `<Grid container>`s.

Reply:
done